src/mesh/test/tstDraco_Mesh_DD.cc

```diff
@@ -20,6 +21,18 @@ using rtt_mesh_test::Test_Mesh_Interface;
 
 // 2D Cartesian domain-decomposed mesh construction test
 void cartesian_mesh_2d_dd(rtt_c4::ParallelUnitTest &ut) {
+  //----------------------------------------------------------------------------------------------//
+  // Two-rank mesh schematic for this test:
+  //    y
+  //    ^
+  //    |
+  //  1 -----------
+  //    |    |    |
+  //    | r0 | r1 |
+  //    |    |    |
+  //  0 ------------> x
+  //    0    1    2
+  //----------------------------------------------------------------------------------------------//
```

Comment:
Nice!

Reply:
Thanks!  These tests have a lot of hard-coded checks on the layouts, which can be difficult to read through.
Until the layout checks are more streamlined, a basic schematic of the decomposition seemed useful.